Automattic/themes
# Iotix: add startup pattern to home.html

#### Changes proposed in this Pull Request:

This PR adds a `startup` pattern to Iotix, and includes it home template. 

This is more consistent with other themes, so that users don't have to fiddle with reading settings to get the demo to match their site.

cc @beafialho 

#### To Test:

- Check out this PR
- Go to the site editor, verify home.html appears as expected
- Go to the front, verify front page appears as expected
